src/components/WeekDayPicker/index.d.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactNode, ChangeEvent } from 'react';
+import { BaseProps } from '../types';
+
+type WeekDays = 'monday' | 'tuesday' | 'wednesday' | 'thursday' | 'friday' | 'saturday' | 'sunday';
+type Value = WeekDays | WeekDays[];
+
+export interface WeekDayPickerProps extends BaseProps {
+    id?: string;
+    name?: string;
+    value?: Value;
+    label?: ReactNode;
+    bottomHelpText?: string;
```

Comment:
bottomHelpText is a ReactNode, we pass all type of text as nodes since there are react internationalization libraries that pass a react node like react intl which pass a component to make the translation:
```<FormattedMessage defaultMessage="my text in english" />```